CONTRIBUTING.md

```diff
@@ -12,6 +12,21 @@ Opening issues will allows us to write better code and make sure it is more usab
 
 If you wish to contribute with a translation of the project in your language, we are of course happy to accept them.
 
+## Steps to contribute
+We are using test to check the code before merging. If you wish open a pull request to fix any bug o enhancement the application,
```

Comment:
"All PR need to pass some tests before they are merged. You can find a list of tests in travis-ci.org.
You can also run the same tests locally to make sure that PR can be accepted. In order to do so, you can execute the following commands"

'Bundle install' installs the gems, there is no need to refer to that.